syntax = "proto3";

service BidirectionalStreaming {
   rpc GetServerResponse(stream Message) returns (stream ServerReply) {}
}

message Message {
  int32 generation = 1;
  float fitness = 2;
  float normalizeFitness = 3;
  repeated int32 weights = 4;
  repeated float delays = 5;
}

message ServerReply {
  int32 status = 1;
}
